src/react-components/media-browser.js

```diff
@@ -346,14 +347,20 @@ class MediaBrowser extends Component {
                   )}
                 {urlSource === "scenes" && (
                   <div className={styles.engineAttributionContents}>
-                    <FormattedMessage id={`media-browser.powered_by.${urlSource}`} />
-                    <a href="/spoke" target="_blank" rel="noreferrer noopener">
-                      <FormattedMessage id="media-browser.spoke" />
-                    </a>
-                    |
-                    <a target="_blank" rel="noopener noreferrer" href="/?report">
-                      <FormattedMessage id="media-browser.report_issue" />
-                    </a>
+                    {configs.feature("enable_spoke") && (
+                      <>
```

Comment:
as an aside, seems we should be locking down the APIs on this flag too. for a later pass ofc